src/lab14-tests.md

```diff
@@ -226,7 +226,7 @@ Tab order causes focus:
 
     >>> browser.tabs[0].advance_tab()
     >>> browser.tabs[0].focus
-    <input value="">
```

Comment:
I don't remember why this was here; why is it now breaking things?

Reply:
Ah—you used to clear `value` on focus, the new code does it on activate (a purposeful change in semantics, the old semantics were very hard to use) so `value` isn't set yet after focusing.